mitmproxy/contentviews.py

```diff
@@ -172,7 +172,7 @@ class ViewHex(View):
     def _format(data):
         for offset, hexa, s in strutils.hexdump(data):
             yield [
-                ("offset", offset + " "),
+                ("offset", offset + b" "),
```

Comment:
The offset is actually `bytes`: https://github.com/mitmproxy/mitmproxy/pull/1299/commits/c7cbbf5f0de9bceccc37b73586ed3baf9421f63a

I didn't know whether to fix that or this!